src/ChestShop/EventListener.php

```diff
@@ -73,6 +74,10 @@ public function onTransaction(InventoryTransactionEvent $event) : void{
 		$playerAction = null;
 
 		foreach($actions as $action){
+			if($action instanceof DropItemAction) {
+				$event->setCancelled(true);
```

Comment:
Why are you cancelling the event completely?

Reply:
@Muqsit Why not?

Reply:
You are cancelling all `DropItemAction` in the process, even when the player isn't viewing the custom inventory.